### DISCUSSION TITLE
Youtube Embed

### DISCUSSION THREAD

**Discussion by saliksik** [satnaing/astro-paper#115] [Q&A]
Hi Guys,

How to properly embed Youtube videos on markdown posts so that it will be responsive on all mobile devices?

Thank you!

**Comment by satnaing** [comment]
You can embed a YouTube video inside a markdown file like this

file: `src/content/blog/your-markdown-file.md`

```md
<!-- Other contents -->

<iframe class="w-full aspect-video" src="https://www.youtube.com/embed/Vi2XaiKhgiU?si=eMs4JJn5cGKIwdiO" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

<!-- Other contents -->
```

All you have to do is remove `width` and `height` from the `iframe` and add `"w-full aspect-video"` class in that `iframe`.

Let me know if you still have some problem.
